packages/shared/ReactServerContextRegistry.js

```diff
@@ -9,21 +9,69 @@
 
 import type {ReactServerContext} from 'shared/ReactTypes';
 
-import {REACT_SERVER_CONTEXT_DEFAULT_VALUE_NOT_LOADED} from 'shared/ReactSymbols';
+import {
+  REACT_PROVIDER_TYPE,
+  REACT_SERVER_CONTEXT_TYPE,
+  REACT_SERVER_CONTEXT_DEFAULT_VALUE_NOT_LOADED,
+} from 'shared/ReactSymbols';
+
 import ReactSharedInternals from 'shared/ReactSharedInternals';
-import {createServerContext} from 'react';
 
 const ContextRegistry = ReactSharedInternals.ContextRegistry;
 
 export function getOrCreateServerContext(
   globalName: string,
 ): ReactServerContext<any> {
   if (!ContextRegistry[globalName]) {
-    ContextRegistry[globalName] = createServerContext(
-      globalName,
-      // $FlowFixMe[incompatible-call] function signature doesn't reflect the symbol value
-      REACT_SERVER_CONTEXT_DEFAULT_VALUE_NOT_LOADED,
-    );
+    const context: ReactServerContext<any> = {
```

Comment:
why is this inlined now?

Reply:
I think that is because we don’t want to warn when this method is called which is used internally in react

Reply:
Yea and I plan on literally deleting the code soon so meh.